File: backend/WebApi/Application/UserNotes/Models/UserNoteModel.cs
using Domain.Entities;
using System;

namespace Application.UserNotes.Models
{
    public sealed class UserNoteModel
    {
        public Guid Id { get; set; }
        public string Text { get; set; }
        public DateTime Created { get; set; }
        public DateTime Сompletion { get; set; }
        public Guid UserId { get; set; }
        public NotePriority Priority { get; set; }
        public bool IsActual { get; set; }
    }
}

File: frontend/MyNotes.Angular/MyNotes.Angular/src/app/models/user-note-model.ts
import { NotePriority } from "../services/clients/my-notes-api.client";

export class UserNoteModel{
    constructor(
        public id: string,
        public text: string,
        public created: Date,
        public сompletion: Date,
        public userId: string,
        public priority: NotePriority,
        public isActual: boolean
    ){}
}